flink-streaming-java/src/test/java/org/apache/flink/streaming/util/AbstractStreamOperatorTestHarness.java

```diff
@@ -309,6 +310,85 @@ public void initializeState(OperatorStateHandles operatorStateHandles) throws Ex
 		initializeCalled = true;
 	}
 
+	/**
+	 * Takes the different {@link OperatorStateHandles} created by calling {@link #snapshot(long, long)}
+	 * on different instances of {@link AbstractStreamOperatorTestHarness} (each one representing one subtask)
+	 * and repacks them into a single {@link OperatorStateHandles} so that the parallelism of the test
+	 * can change arbitrarily (i.e. be able to scale both up and down).
+	 * <p/>
```

Comment:
A new paragraph should start with `<p>`. `<p/>` should never be used in Javadoc, AFAIK.